awx/settings/defaults.py

```diff
@@ -1151,5 +1151,12 @@
 # Temporary, for old roles API compatibility, save child permissions at organization level
 ANSIBLE_BASE_CACHE_PARENT_PERMISSIONS = True
 
+# Currently features are enabled to keep compatibility with old system, except custom roles
+ANSIBLE_BASE_ALLOW_TEAM_ORG_ADMIN = False
+# ANSIBLE_BASE_ALLOW_CUSTOM_ROLES = True
+ANSIBLE_BASE_ALLOW_CUSTOM_TEAM_ROLES = False
+ANSIBLE_BASE_ALLOW_SINGLETON_USER_ROLES = True
```

Comment:
Shoot, I think I coded myself into a corner here. We need the flag `True` so that it uses it in evaluations. But we really don't want to allow creating these through the API until the UI can support it. So maybe we have to introduce another setting for that.